### DISCUSSION TITLE
minio healing too slow

### DISCUSSION THREAD

**Discussion by hert525** [minio/minio#16872] [Q&A]
![image](https://user-images.githubusercontent.com/19421393/226888803-b2ec6c14-469e-48bb-bc27-61f0faf1f345.png) 
how can i fix it?

**Comment by vadmeste** [comment]
@hert525 what version of MinIO is this ?

**Reply by hert525** [reply]
2021-11-03 tks

**Reply by klauspost** [reply]
Upgrade.

**Reply by hert525** [reply]
2021-11-03 upgrade to 2023-03-23, is there any compatibility issue?

**Reply by harshavardhana** [reply]
@hert525 no you can upgrade normally.
